index.md

```diff
@@ -52,3 +203,61 @@ which most in the community are members of.
 If you have technical questions around DNS flag day you can join
 [the DNS-operations mailing list](https://lists.dns-oarc.net/mailman/listinfo/dns-operations)
 and ask them there.
+
+Get in touch
+============
+
+For press & media inquiries please use media (at) dns-oarc.net and please put
+“DNS Flag Day" in the email subject line.
+
+- Web https://dnsflagday.net/
+- Twitter https://twitter.com/dnsflagday
+- Announcements: https://lists.dns-oarc.net/mailman/listinfo/dns-announce
+- Discussion: https://lists.dns-oarc.net/mailman/listinfo/dns-operations
+
+FAQ
+===
+
+- Q: Is DNS over UDP dead?
+
+  A: No, UDP will still be the main means of transportation.
+
+- Q: TL;DR [RFC 7766](https://tools.ietf.org/html/rfc7766)
+
+  A: DNS **MUST** work over TCP!
```

Comment:
The Internet specifications have ensured for decades that independently-developed systems can interoperate reliably over the open public Internet. These specification call out features that MUST, SHOULD, and MUST NOT work for universal interoperability. DNS over TCP has been agreed upon as a feature that **MUST** be supported. A very small percentage of the Internet is not meeting this requirement, which is causing trouble for the majority, who are respecting the established standards.

Reply:
This was a TL;DR, answers should be very short.